testing/py3-clint/APKBUILD

```diff
@@ -0,0 +1,24 @@
+# Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
```

Comment:
Patch for py3-clint: http://ix.io/1TXS

- Enables tests with py3-pytest (command extracted from their tox.ini)
- Sets noarch